Automattic/wp-calypso
# Marketplace Reviews: Show the user review first on the list

Related to https://github.com/Automattic/wp-calypso/pull/85496

## Proposed Changes

Always display the user review first on the list.
Two requests are being made:
* User reviews for a given product (limited to 1)
* Reviews excluding the user

The lists will be concatenated, keeping this way the user review always as first.

## Testing Instructions

* On this branch
* Go to a plugin details page where you have a review. Ex: `/plugins/woocommerce-bookings/`
* On the bottom, click on Read all reviews
* Check if your review is being displayed first

![CleanShot 2023-12-20 at 08 37 14@2x](https://github.com/Automattic/wp-calypso/assets/5039531/5bb1f413-6a82-4dc9-99b2-e3b44bc07ccd)


## Pre-merge Checklist
